pylint/checkers/base/comparison_checker.py

```diff
@@ -89,16 +96,10 @@ def _check_singleton_comparison(
         checking_for_absence: bool = False,
     ) -> None:
         """Check if == or != is being used to compare a singleton value."""
-        singleton_values = (True, False, None)
-
-        def _is_singleton_const(node: nodes.NodeNG) -> bool:
```

Comment:
If it's used only there it's good that it is an inner function, it's hard to know if you can remove a class function when it's no longer used.

Reply:
I use it in my `magic_number,py` checker, I didn't won't to duplicate the function. what should I do then?

Reply:
My bad, I missed that, how about we keep the function but inline the constant ?